torch/csrc/python_serialization.h

```diff
@@ -0,0 +1,98 @@
+#pragma once
+#include <Python.h>
+
+// Call Python fildes.read(nbytes) and copy it to buf.
+// Requires a buffer for the bytes object returned from read
+// so the reading is chunked.
```

Comment:
The reading is chunked not because we need the buffer. We're enforcing the chunking ourselves. Theoretically we could have just called `f.read(nbytes)` and copy that into `buf`.